src/DependencyInjection/VichUploaderExtension.php

```diff
@@ -4,13 +4,15 @@
 
 use Doctrine\Common\Annotations\AnnotationReader;
 use Symfony\Component\Config\FileLocator;
+use Symfony\Component\CssSelector\Parser\Reader;
```

Comment:
Wrong class imported here. Should be `Doctrine\Common\Annotations\Reader`.

Reply:
My bad, fixed